test/services/cloudsearchdomain.spec.coffee

```diff
@@ -49,8 +49,16 @@ describe 'AWS.CloudSearchDomain', ->
       params = { query: 'foo' }
       req = build('search', params)
       expect(req.headers).not.to.have.property('Authorization')
-    
+
     it 'signs request if credentials are provided', ->
       params = { query: 'foo' }
       req = build('search', params)
       expect(req.headers).to.have.property('Authorization')
+
+    it 'converts the GET request to POST for search operation', ->
```

Comment:
We should probably have the opposite test case as well, where we ensure an operation that is not `search` is still using a `GET` http method.